content/en/blog/_posts/2023-08-15-sidecar-kep.md

```diff
@@ -0,0 +1,113 @@
+---
+layout: blog
+title: "Kubernetes v1.28: Introducing native sidecar containers"
+date: 2023-08-15
```

Comment:
Hi @matthyx , here Comm shadow 1.28, following the blog date publication, you can check it here: https://github.com/orgs/kubernetes/projects/140/views/6

```suggestion
date: 2023-08-25
```

Reply:
You should rename the file to match the new date, too.

Reply:
Do we have to merge this blog on 08-25?